assets/js/content-helps.js

```diff
@@ -0,0 +1,46 @@
+(function($) {
+  function changeHelpButtonState($helpButton, state) {
+    const helpButtonClasses = $helpButton[0].classList
+
+    helpButtonClasses.toggle('selected', state)
+    helpButtonClasses.toggle('ico-after', state)
+    helpButtonClasses.toggle('tick', state)
+    helpButtonClasses.toggle('green', state)
+
+    $helpButton.attr('data-activated', state.toString())
+    $helpButton.blur()
+
+    $helpButton.parent().find('input[name="activated"]').val((!state).toString())
+  }
+
+  $('.help-toggle').click((e) => {
+    e.preventDefault()
+
+    const $current = $(e.target)
+    const $form = $current.parent()
+    const newActivation = $current.attr('data-activated') !== 'true'
```

Comment:
Et ici aussi (ahhh je suis mauvais)

C'est bizarre, c'est comme si la moitié avait été appliqué

```suggestion
    const $current = $(e.target)
    const $form = $current.parent()
    const data = $form.serialize()

    const newActivation = $current.attr('data-activated') !== 'true'
```

Reply:
y'a des trucs que j'avais enlevé car il y avait des variables non utilisées, notamment data.